socorro/external/es/super_search_fields.py

```diff
@@ -14,6 +14,19 @@
 from socorrolib.lib import datetimeutil, external_common
 
 
+ES_CUSTOM_ANALYZERS = {
+    "analyzer": {
+        "semicolon_keywords": {
+            "type": "pattern",
+            "pattern": ";"
+        }
```

Comment:
call me silly but there's a mix of `"` and `'` here. This one uses `"` the `ES_QUERY_SETTINGS` uses `'`.